incidents/2021-06-07-elifesciences-org-cannot-be-resolved.md

```diff
@@ -0,0 +1,69 @@
+# 2021-06-07 Unable to Resolve elifesciences.org
+
+**Incident Leader: Joel Summerfield**
+
+## Description
+
+After using builder to create an ad hoc instance of Journal, which failed, the rollback failed to recreate a DNS A record for elifesciences.org and as such nobody could access the journal using the primary address.
+
+## Timeline
+
+**2021-06-07**
+
+- 12:03 Joel runs the builder command `./bldr launch:journal,mathjaxv3,prod` to create an ad hoc ec2 instance of journal with a production configuration. The command fails, and builder automatically performs a rollback.
```

Comment:
> builder automatically performs a rollback

incorrect, AWS Cloudformation performed the rollback, builder generates the change, triggers it and monitors it's progress. Builder doesn't contain any code to trigger rollbacks.